R/greta_model_class.R

```diff
@@ -200,6 +200,8 @@ print.greta_model <- function (x, ...) {
 
 #' @rdname model
 #' @param y unused default argument
+#' @param color base color used for plotting. Defaults to \code{greta} colors
```

Comment:
could you please change 'color' -> 'colour', the rest of the package uses UK/Australian spelling, so it would be good to be consistent